demo/docker_manufacturer/README.md

```diff
@@ -47,6 +47,12 @@ This is an example implementation for demo purposes and should be updated in pro
 Modify mt_config.sql and rt_config.sql following instructions given in the respective files to customize for your environment,
 and replace the sample keystore.
 
+### OnDie ECDSA configuration
+
+If OnDie ECDSA support is needed, run the script located at <sdo-supply-chain-tools>/scripts/onDieCache.py,
```

Comment:
Provide a hyperlink to https://github.com/secure-device-onboard/supply-chain-tools/blob/master/scripts/onDieCache.py